### ISSUE TITLE
Site not pulling data

### ISSUE THREAD

**Issue by HyperData3434** [d-m-wilson/HoustonIncidents#1]
The web GUI isn't pulling data from HPD or HFD, it's only TransStar events shown on the map and in the archive search

**Comment by HyperData3434** [comment]
Attempting to fix it now, will add a pull request if I manage to fix it

**Comment by d-m-wilson** [comment]
The active incidents web service has been down for some time. https://hfdapp.houstontx.gov/JSONTest/WebService1.asmx?op=GetIncidents I have no control over it, and I'm not sure who does. I have reached out to a few people who work for the City of Houston, but no luck yet.

**Comment by HyperData3434** [comment]
Yeah, after I dug for a bit I figured that was the case. Massive bummer, this was an awesome tool.

**Comment by commandrea** [comment]
It looks like the active incidents page is back up - not sure if that's helpful [https://cohweb.houstontx.gov/ActiveIncidents/Combined.aspx](https://cohweb.houstontx.gov/ActiveIncidents/Combined.aspx)

**Comment by d-m-wilson** [comment]
Unfortunately, I have never been able to use that page as it does not have latitude and longitude. Nor are the addresses sufficient to geocode them. The old SOAP web service included (what I assumed are) approximate lat/long coordinates - something easy to plot on a map. I'm afraid that if the web service can't be fixed or replaced somehow, then this project will no longer have a future. There is now a desperate plea on the page where my Active Incidents map used to be. ☹️

**Comment by HyperData3434** [comment]
I do not know how realistic this is, but there is a live feed of the HFD dispatch frequency, that includes the address of the calls, I can work on a project that is able to translate the audio into a csv or something that can then be fed into your map

**Comment by HyperData3434** [comment]
[Here's the link to the project, I'll put any updates there](https://github.com/HyperData3434/HFD-Incident-Pull)

**Comment by d-m-wilson** [comment]
Someone contacted me with a new source for what appears to be the same incident data. (Thanks, Hans!) The City of Houston has public-facing ESRI ArcGIS REST endpoints. https://mycity2.houstontx.gov/pubgis01/rest/services/HEC/HEC_Active_Incidents/MapServer/0/

**Comment by HyperData3434** [comment]
Amazing!! If you need any help, just let me know!

**Comment by d-m-wilson** [comment]
I deployed some new code today. Y'all please let me know if things don't seem to work right.

Edit: I'll update this repo at some point. 😉 I wanted to get the site back up ASAP.

**Comment by HyperData3434** [comment]
All appears to be functional! I'll reply to this issue or add a new one if it fixes it

**Comment by HyperData3434** [comment]
It's still up as of now, I haven't seen any issues!
Just wanted to give an update after this has been out for a while
